### PR TITLE
Refactoring the pipeline for incremental scripts - I

### PR DESCRIPTION
**Pull request by covid19owid**
@ValentinMouret and @edomt 

Pipeline for couple of incremental scripts have been updated to Series. Can you please provide your reviews, comments and feedback.

### CODE REVIEW

File: scripts/scripts/vaccinations/automations/incremental/portugal.py [hunk 1]
```diff
@@ -1,29 +1,73 @@
...
+
+
+def parse_people_fully_vaccinated(data: pd.DataFrame) -> str:
+    people_fully_vaccinated = data["Inoculacao2_Ac"]
+    return people_fully_vaccinated
+
+
+def parse_people_vaccinated(data: pd.DataFrame) -> str:
+    people_vaccinated = data["Inoculacao1_Ac"]
+    return people_vaccinated
```
Comment (ValentinMouret) [inline scripts/scripts/vaccinations/automations/incremental/portugal.py:37]:
```suggestion
def parse_total_vaccinations(data: dict) -> int:
    return int(data["Vacinados_Ac"])


def parse_people_fully_vaccinated(data: dict) -> int:
    return int(data["Inoculacao2_Ac"])


def parse_people_vaccinated(data: dict) -> int:
    return int(data["Inoculacao1_Ac"])
```

**Review by ValentinMouret — APPROVED** [review]
Thanks! I made some suggestions.

**Comment by covid19owid** [comment]
@ValentinMouret . Thanks a lot !. Appreciate your assistance as always.

**Comment by covid19owid** [comment]
@ValentinMouret and @edomt 

Thank you again very much for the reviews and comments. The suggested changes have been made to the files.

**Comment by edomt** [comment]
Thank you!

**Pull request merged**